src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Subscriptions/DependencySubscriptionsHost.cs

```diff
@@ -270,13 +265,14 @@ private void OnSubtreeProviderDependenciesChanged(object sender, DependenciesCha
 
             ImmutableDictionary<ITargetFramework, IDependenciesChanges> changes = ImmutableDictionary<ITargetFramework, IDependenciesChanges>.Empty.Add(targetFramework, e.Changes);
 
-            UpdateDependenciesSnapshotAsync(changes, catalogs: null, activeTargetFramework: null);
+            UpdateDependenciesSnapshotAsync(changes, catalogs: null, activeTargetFramework: null, e.Token);
         }
 
         private void UpdateDependenciesSnapshotAsync(
             ImmutableDictionary<ITargetFramework, IDependenciesChanges> changes,
             IProjectCatalogSnapshot catalogs,
-            ITargetFramework activeTargetFramework)
+            ITargetFramework activeTargetFramework,
+            CancellationToken token = default)
```

Comment:
I don't like making these optional for things that aren't part of a public API.  It's too easy to avoid passing them along.

Reply:
Updated.